src/test/ui/consts/miri_unleashed/feature-gate-unleash_the_miri_inside_of_you.stderr

```diff
@@ -2,7 +2,9 @@ error[E0493]: destructors cannot be evaluated at compile-time
   --> $DIR/feature-gate-unleash_the_miri_inside_of_you.rs:11:20
    |
 LL |     const F: u32 = (U::X, 42).1;
-   |                    ^^^^^^^^^^ constants cannot evaluate destructors
+   |                    ^^^^^^^^^^ - value is dropped here
```

Comment:
This is a slightly odd span. It points to the field index. I mean it's kind of correct, the field access is where it gets dropped... Probably fine to leave it as it is